gdal/ogr/ogrsf_frmts/pgeo/ogrpgeodatasource.cpp

```diff
@@ -232,17 +232,45 @@ int OGRPGeoDataSource::Open( const char * pszNewName, int bUpdate,
                         && osLCTableName != "msysaces"
                         && osLCTableName != "msysqueries"
                         && osLCTableName != "msysrelationships"
+                        && osLCTableName != "gdb_annosymbols"
```

Comment:
This list begins to be seriously painful to maintain. Why not just do ``&& !(osLCTableName.size() >= 4 && osLCTableName.substr(0, 4) == "gdb_")`` ?